src/product-attributes/product-attributes.service.ts

```diff
@@ -92,9 +88,12 @@ export class ProductAttributesService {
       }
 
       updateProductAttributeData.options = updateProductAttributeDto.options;
+
     }
-    
-    return this.prisma.productAttribute.update({
+    else if (updateProductAttributeDto.type === 'text') {
```

Comment:
since there is no extra condition after this else if why are we using else if instead of just else